index.js

```diff
@@ -50,7 +60,16 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies) {
+  if(movies.length===0){
+    throw "Error Empty Array"
+  }
+
+  let allratings = movies.some(movie => movies.rated )
```

Comment:
```suggestion
  let allratings = movies.some(movie => movie.rated === rating )
```
compare the movie's rating to the inputted rating. 
It should be "movie.rated" instead of "movies.rated" because "movies" is the whole array. "movie" is each single object in the array.